hicommonwealth/commonwealth
# Bridged forum webhook options

## Link to Issue
Closes: #4961 

## Description of Changes
- Allows admins to toggle on/off webhook notifications for posts that come from the Discobot. 

## "How We Fixed It"

Introduced a new column on chains to store this setting, updated UI to handle it and updated webhook notifier to handle discobot posts accordingly.

## Test Plan
1. Ensure `TELEGRAM_BOT_TOKEN_DEV`, `DISCORD_WEBHOOK_URL_DEV`, `ZAPIER_WEBHOOK_URL_DEV`, `SLACK_WEBHOOK_URL_DEV` are set in the environment. See `Webhooks.md` in the wiki for information on where to get those (or message @timolegros on Slack)
2. Run `yarn emit-webhook -c [category] -d all` once for each category: chain-event, new-thread-creation, new-comment-creation, new-reaction. Between each execution ensure each message in Discord, Slack, and Telegram has the correct formatting and links.

## Deployment Plan

1. 

## Other Considerations

-
